src/utils/ipfsUtils.ts

```diff
@@ -106,3 +76,21 @@ export const getStageData = async (
 
   return data;
 };
+
+const getIpfsClient = async (): Promise<IPFSHTTPClient> => {
+  const auth =
```

Comment:
This seems like it is being called many times unecessarily.
Can it be cached and reused?

Reply:
Fixed